src/webgpu/shader/execution/expression/call/builtin/texture_builtin_notes.md

```diff
@@ -0,0 +1,918 @@
+# Texture Builtin Notes
+
+These are known failures for various GPUs. If you find more, please add to this list so developers
+can be aware what does and does not work.
```

Comment:
If we want this to be discoverable by developers then maybe it should be on some github wiki or something. I'm not sure where is a good place for it.

Otherwise having it here is fine, though I'd add a mention of this file in one of the READMEs or test description files so it's not "unreferenced" from the test suite.

Reply:
wiki works for me.

Reply:
I started a wiki here

https://github.com/gpuweb/gpuweb/wiki/Texture-Builtins-Issues

I thought about making one page per issue and linking them from the that page. Or I can just past them all in that one page.

Reply:
Personally I think one page is better, github wikis don't really handle having many pages that well.

As for this PR, it'll still useful to add a link to that wiki page in one of the readmes/descriptions.

Reply:
I posted the issue is the wiki but unfortunately it's not really readable as github gives the wiki a maximum width. Maybe the wiki should just link to this and/or, maybe we could make it available at `https://gpuweb.github.io/cts/standalone/src/webgpu/shader/execution/expression/call/builtin/texture_builtin_notes.md`

Reply:
Ah, shame.

Linking is fine, we could have a gpuweb wiki page for errata, and link directly to the .md file in this repo. Don't think there's any need to make it work on gpuweb.github.io.